truffle-config.js

```diff
@@ -32,6 +32,15 @@ module.exports = {
       network_id: '*', // match any network
       gasPrice: '0'
     },
+    bordevnet: {
+      provider: () =>
+        new HDWalletProvider(
+          MNEMONIC,
+          `http://3.94.19.25:9545`
```

Comment:
what is it for? Why new network is used?

Reply:
It was used for safemath deployment test.